src/System.Management.Automation/utils/StructuredTraceSource.cs

```diff
@@ -924,6 +924,61 @@ internal void WriteLine(string format, object arg1)
             }
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
PowerShellGet - Module tests.Should install a module correctly to the required location with default CurrentUser scope
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/PowerShellGet/PowerShellGet.Tests.ps1: line 134
134:         $installedModuleInfo | Should -Not -BeNullOrEmpty
```